src/examples/tags/TagsSize.tsx

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import { Span } from '@zendeskgarden/react-typography';
+import { Tag } from '@zendeskgarden/react-tags';
+import { Row, Col } from '@zendeskgarden/react-grid';
+
+const Example = () => (
+  <Row>
```

Comment:
For consistency with https://garden.zendesk.com/components/button#size, add `alignItems="center"` to the Row.